systemtest/src/test/java/io/strimzi/systemtest/rollingupdate/KafkaRollerST.java

```diff
@@ -266,6 +275,13 @@ void testKafkaPodPendingDueToRack() {
         KafkaUtils.waitForKafkaReady(CLUSTER_NAME);
     }
 
+    boolean checkIfOneKafkaPodIsNotReady(String clusterName) {
```

Comment:
`checkIfExactlyOneKafkaPodIsNotReady` would be clearer (otherwise it might be mistaken for an 'at least one' condition)